web/json_value.go

```diff
@@ -192,6 +202,15 @@ func ParseDateTime(dateTimeFormats []DateTimeFormatWithTimeZone, localTimeZoneOf
 				)
 			}
 
+			// Convert unix timestamp to rfc3339 format
+			if format.Format == "sgnl-unix-ts" {
+				rfc3339Time := dateTime.Format(time.RFC3339)
+				dateTime, err = time.Parse(time.RFC3339, rfc3339Time)
+				if err != nil {
+					err = fmt.Errorf("panic: unix post conversion to rfc3339 failed. dateTimeStr: %s, parse error: %v", dateTimeStr, err)
+				}
+			}
```

Comment:
Why do we need this? [time.Unix](https://pkg.go.dev/time#Unix) returns `time.Time`.